suite/cstest/include/test_detail_bpf.h

```diff
@@ -41,6 +43,10 @@ static const cyaml_schema_field_t test_detail_bpf_op_mapping_schema[] = {
 			       TestDetailBPFOp, mem_base, 0, CYAML_UNLIMITED),
 	CYAML_FIELD_INT("mem_disp", CYAML_FLAG_OPTIONAL, TestDetailBPFOp,
 			mem_disp),
+	CYAML_FIELD_BOOL("is_pkt", CYAML_FLAG_OPTIONAL, TestDetailBPFOp,
+			 is_pkt),
+	CYAML_FIELD_BOOL("is_signed", CYAML_FLAG_OPTIONAL, TestDetailBPFOp,
+			 is_signed),
```

Comment:
```suggestion
	CYAML_FIELD_INT("is_pkt", CYAML_FLAG_OPTIONAL, TestDetailBPFOp,
			 is_pkt),
	CYAML_FIELD_INT("is_signed", CYAML_FLAG_OPTIONAL, TestDetailBPFOp,
			 is_signed),
```

Reply:
Why change it to INT?

Reply:
Actually, it would be nice to add a macro for this case.
Can you add `#define CYAML_FIELD_TBOOL CYAML_FIELD_INT` in a common header please?

Reply:
The MacOS test fails likely because of it:
```
2024-12-09T13:41:07.3611390Z INFO  - Run test: TestInput { arch: bpf, options: [
'CS_OPT_DETAIL', 'CS_MODE_BPF_EXTENDED'], addr: 0, bytes: [ 0x28,0x00,0x00,0x00,
0xff,0x00,0x00,0xfa,0x07,0x76,0x01,0x28,0xff,0xff,0xff,0xff,0x85,0xd3,0xa5,0xe2,
0xff,0xff,0xff,0xff ] }
2024-12-09T13:41:07.3612030Z WARNING - CS_ARCH: Capstone doesn't have the attrib
ute 'bpf'
2024-12-09T13:41:07.3612920Z ERROR - is_pkt: False != True
2024-12-09T13:41:07.3613090Z INFO  - TestResult.FAILED
```